src/Model/GroundWaterFlow/gwf3drn8.f90

```diff
@@ -18,6 +20,7 @@ module DrnModule
   !
   character(len=LENFTYPE) :: ftype = 'DRN'
   character(len=LENPACKAGENAME) :: text = '             DRN'
+  character(len=LENMEMSEPARATOR), parameter :: memPathSeparator = '/'
```

Comment:
This is already defined in MemoryHelperModule.  You should USE it instead.

Reply:
Will do.  I'll fix it GHB and RIV as well.

Reply:
Addressed in 835699c